Classes/Service/ContentService.php

```diff
@@ -423,12 +433,25 @@ protected function getPreviousLocalizedRecordUid($uid, $language, DataHandler $r
             // Respect the colPos for content elements
             if ($table === 'tt_content') {
                 $where .= sprintf(
-                    ' AND colPos=%d AND tx_flux_column=\'%s\' AND tx_flux_parent=%d',
-                    (integer) $row['colPos'],
-                    $row['tx_flux_column'],
+                    ' AND colPos=%d',
```

Comment:
This and the next changes to build $where looks unnecessary: the previous code results in the same output as these conditions as the values that get used match the default values of fields.

Reply:
The problem is when $row['tx_flux_column'] or $row['tx_flux_parent'] is NULL